satellite/console/consoleweb/server.go

```diff
@@ -320,19 +369,24 @@ func (server *Server) cancelPasswordRecoveryHandler(w http.ResponseWriter, r *ht
 	ctx := r.Context()
 	defer mon.Task()(&ctx)(nil)
 	recoveryToken := r.URL.Query().Get("token")
-	if len(recoveryToken) == 0 {
-		http.Redirect(w, r, "https://storjlabs.atlassian.net/servicedesk/customer/portals", http.StatusSeeOther)
-	}
 
 	// No need to check error as we anyway redirect user to support page
 	_ = server.service.RevokeResetPasswordToken(ctx, recoveryToken)
 
+	// TODO: Should place this link to config
 	http.Redirect(w, r, "https://storjlabs.atlassian.net/servicedesk/customer/portals", http.StatusSeeOther)
 }
 
-func (server *Server) serveError(w http.ResponseWriter, r *http.Request) {
-	w.WriteHeader(http.StatusNotFound)
-	http.ServeFile(w, r, filepath.Join(server.config.StaticDir, "static", "errors", "404.html"))
+func (server *Server) serveError(w http.ResponseWriter, r *http.Request, status int) {
+	// TODO: show different error pages depend on status
+	// F.e. switch(status)
+	//      case http.StatusNotFound: server.executeTemplate(w, r, notFound, nil)
+	//      case http.StatusInternalServerError: server.executeTemplate(w, r, internalError, nil)
+	w.WriteHeader(status)
+
+	if err := server.templates.pageNotFound.Execute(w, nil); err != nil {
+		server.log.Error("error occurred in console/server", zap.Error(err))
```

Comment:
You don't need to mention the package where the error occurs, since the `server.log` already takes care of it. Here the appropriate message would be `"unable to execute pageNotFound", zap.Error(err)`